pyte/streams.py

```diff
@@ -231,7 +245,7 @@ def _parser_fsm(self):
                                   ctrl.VT, ctrl.FF, ctrl.CR])
         OSC_TERMINATORS = set([ctrl.ST_C0, ctrl.ST_C1, ctrl.BEL])
 
-        def create_dispatcher(mapping):
+        def create_dispatcher(mapping: dict[str, str]) -> dict[str, Callable[..., None]]:
```

Comment:
Consider using typing.Mapping instead of a dict.

Reply:
I think `Mapping` is better suited for public functions that need to be lax in what they accept (be lax in what you accept and strict in what you admit).  But, I can change it.

Reply:
I switched to `Mapping`, and fixed other instances where I was using generic builtins that aren't available in older pythons.